addons/resource.language.en_gb/resources/strings.po

```diff
@@ -22852,3 +22851,10 @@ msgstr ""
 msgctxt "#39174"
 msgid "Display supported HDR types"
 msgstr ""
+
+#. Window screen calibration
+#: addons/skin.estuary/xml/SettingsScreenCalibration.xml
```

Comment:
I don't think this is correct. Addon/skin specific localized values should be handled by the addon translations, not core.

Reply:
Or at least, if this is a core string, the xml in the skin should not be listed here as it is not part of core

Reply:
this made me weird too, so i waited for feedback from you,
all ID's of "calibration window" are here on core, i will move this new ID to addon skin localization,
since i'm going to have to do a little code cleanup PR i could move all others ID's to addon skin localization